app/services/invitation_service.rb

```diff
@@ -27,11 +27,20 @@ def self.invite_to_group(recipient_emails: nil,
                                                inviter: inviter)
 
       InvitePeopleMailer.delay.to_join_group(invitation: invitation,
-                                             locale: I18n.locale)
+                                             locale: I18n.locale,
+                                             subject: "email.to_join_group.subject")
       invitation
     end
   end
 
+  def self.resend(invitation)
+    return if invitation.accepted_at || invitation.cancelled_at || !invitation.send_count
```

Comment:
I think this should be moved into the invitation model

```
return unless invitation.is_pending?
```

Also .. why the sent_count check?